app/models/manageiq/providers/openstack/cloud_manager/volume_snapshot_template.rb

```diff
@@ -0,0 +1,12 @@
+class ManageIQ::Providers::Openstack::CloudManager::VolumeSnapshotTemplate < ManageIQ::Providers::CloudManager::Template
+  # VolumeSnapshotTemplates are proxies to allow provisioning instances from volumes
+  # without having to refactor the entire provisioning workflow to support types
+  # other than VmOrTemplate subtypes. VolumeSnapshotTemplates are created 1-to-1 during
+  # inventory refresh for each eligible snapshot.
+
+  belongs_to :cloud_tenant
+
+  def volume_snapshot_template?
```

Comment:
Where is this used?  Better to just use `.kind_of?(klass)` I think

Reply:
These are in line with `image?` and `snapshot?` which are present on the parent class and used for differentiating type in the UI.

Reply:
:+1: thanks